profile/Makefile

```diff
@@ -34,14 +34,25 @@ Timer.cpp \
 Track.cpp \
 TrackGenerator.cpp \
 Universe.cpp \
-Vector.cpp
+Vector.cpp \
+\
```

Comment:
@geogunow We use Geometry::findNextCell in MCSolver.cpp in the code I copied from the segmentize routine. It's used in finding the next cell the neutron will hit along its path.